library/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksStep.java

```diff
@@ -13,12 +13,7 @@
 import bio.terra.landingzone.stairway.flight.exception.ResourceCreationException;
 import bio.terra.stairway.FlightContext;
 import com.azure.core.management.exception.ManagementException;
```

Comment:
IntelliJ made this decision (to do * instead of adding 3 more import statements)

Reply:
Thanks for pointing this out!

I generally consider wildcard imports to be [harmful](https://errorprone.info/bugpattern/WildcardImport#:~:text=Wildcard%20imports%20are%20forbidden%20by,for%20programmers%20and%20for%20tools.), so I'd recommend setting this in Intellij to 999: https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Reply:
ok, I briefly doubted myself but reverted it back :) I also agree * should be avoided!